Layout.py

```diff
@@ -75,7 +91,10 @@ def load_layout_from_json(self, layout):
         self.fixed_set_layout(layout)
 
         def dict_to_view(view_dict, group, index):
-            view = self.window.open_file(view_dict['file_name'])
+            try:
+                view = self.window.open_file(view_dict['file_name'])
+            except TypeError:
+                return
```

Comment:
added this check to prevent an error message showing up in the output with unsaved files